Server_Side_TLS.mediawiki

```diff
@@ -262,15 +262,12 @@ MBYCEQCHU6UNZoHMF6bPtj21Hn/bAgEC.....
 </source>
 
 == Pre-defined DHE groups ==
-In order to lower the burden of system administrators, several servers provide pre-computed DH groups. Unfortunately, the [https://weakdh.org/ logjam] report showed that it is very likely that a state-level adversary may have broken the most widely used 1024-bit DH group, Oakley group 2, standardized in [https://tools.ietf.org/html/rfc2409#section-6.2 rfc2409]].
+In order to lower the burden of system administrators, several servers provide pre-computed DH groups. Unfortunately, the [https://weakdh.org/ logjam] shows that pre-defined 1024 bits DHE is not safe, and administrators should either:
```

Comment:
Would you say it's unreasonable advice to recommend operators to run `openssl dhparam 2048`, versus using preconfigured groups? The downside being web servers don't allow operators to configure a given group, so if dhparam isn't set by the ops, there's a risk of using a weak group.

I think the computational load is irrelevant here because only done once when configuring the service, or at regular interval, but definitely not during the handshake.